cmd/peggo/flags.go

```diff
@@ -10,39 +10,40 @@ const (
-	flagCosmosGRPC           = "cosmos-grpc"
-	flagTendermintRPC        = "tendermint-rpc"
-	flagCosmosGasPrices      = "cosmos-gas-prices"
-	flagCosmosKeyring        = "cosmos-keyring"
-	flagCosmosKeyringDir     = "cosmos-keyring-dir"
-	flagCosmosKeyringApp     = "cosmos-keyring-app"
-	flagCosmosFrom           = "cosmos-from"
-	flagCosmosFromPassphrase = "cosmos-from-passphrase"
-	flagCosmosPK             = "cosmos-pk"
-	flagCosmosUseLedger      = "cosmos-use-ledger"
-	flagEthKeystoreDir       = "eth-keystore-dir"
-	flagEthFrom              = "eth-from"
-	flagEthPassphrase        = "eth-passphrase"
-	flagEthPK                = "eth-pk"
-	flagEthUseLedger         = "eth-use-ledger"
-	flagEthRPC               = "eth-rpc"
-	flagEthGasAdjustment     = "eth-gas-price-adjustment"
-	flagEthAlchemyWS         = "eth-alchemy-ws"
-	flagRelayValsets         = "relay-valsets"
-	flagRelayBatches         = "relay-batches"
-	flagCoinGeckoAPI         = "coingecko-api"
-	flagEthGasPrice          = "eth-gas-price"
-	flagEthGasLimit          = "eth-gas-limit"
-	flagPowerThreshold       = "power-threshold"
-	flagAutoApprove          = "auto-approve"
-	flagOrchLoopDuration     = "orch-loop-duration"
-	flagEthBlocksPerLoop     = "eth-blocks-per-loop"
-	flagEthPendingTXWait     = "eth-pending-tx-wait"
+	flagLogLevel                = "log-level"
+	flagLogFormat               = "log-format"
+	flagSvcWaitTimeout          = "svc-wait-timeout"
+	flagAutoConfirm             = "yes"
+	flagCosmosChainID           = "cosmos-chain-id"
+	flagCosmosGRPC              = "cosmos-grpc"
+	flagTendermintRPC           = "tendermint-rpc"
+	flagCosmosGasPrices         = "cosmos-gas-prices"
+	flagCosmosKeyring           = "cosmos-keyring"
+	flagCosmosKeyringDir        = "cosmos-keyring-dir"
+	flagCosmosKeyringApp        = "cosmos-keyring-app"
+	flagCosmosFrom              = "cosmos-from"
+	flagCosmosFromPassphrase    = "cosmos-from-passphrase"
+	flagCosmosPK                = "cosmos-pk"
+	flagCosmosUseLedger         = "cosmos-use-ledger"
+	flagEthKeystoreDir          = "eth-keystore-dir"
+	flagEthFrom                 = "eth-from"
+	flagEthPassphrase           = "eth-passphrase"
+	flagEthPK                   = "eth-pk"
+	flagEthUseLedger            = "eth-use-ledger"
+	flagEthRPC                  = "eth-rpc"
+	flagEthGasAdjustment        = "eth-gas-price-adjustment"
+	flagEthAlchemyWS            = "eth-alchemy-ws"
+	flagRelayValsets            = "relay-valsets"
+	flagRelayBatches            = "relay-batches"
+	flagCoinGeckoAPI            = "coingecko-api"
+	flagEthGasPrice             = "eth-gas-price"
+	flagEthGasLimit             = "eth-gas-limit"
+	flagPowerThreshold          = "power-threshold"
+	flagAutoApprove             = "auto-approve"
+	flagOrchLoopDuration        = "orch-loop-duration"
+	flagEthBlocksPerLoop        = "eth-blocks-per-loop"
+	flagEthPendingTXWait        = "eth-pending-tx-wait"
+	flagRelayerProfitMultiplier = "relayer-profit-multiplier"
```

Comment:
nit: wdyt? 

```suggestion
	flagProfitMultiplier = "profit-multiplier"
```

Reply:
Agree, there's no other profit to get confused about. I'll change that 👌